### ISSUE TITLE
Not working in chrome due to chrome issue

### ISSUE THREAD

**Issue by tomermes** [brunobar79/J-I-C#42]
As noted in the following question:

http://stackoverflow.com/questions/20278017/chrome-todataurl-quality-param-is-not-effect-when-export-jpeg

the toDataURL quality param has no effect in chrome and thus this library has no effect too.

**Comment by liltinkerer** [comment]
Indeed. PNG files just get bigger instead of compression. Tested on Chrome.

**Comment by skinTest** [comment]
in the demo page, the 2.7 mb after process will grow to 3.1 mb, which drives me crazy

**Comment by SoreGums** [comment]
PNG needs an option to change its kind to PNG-8, usually good enough for screenshots, which is what I wanted to use this library for.

**Comment by hishosho** [comment]
canvas.toDataURL(type, encoderOptions): encoderOptions only can used to image/jpeg or image/webp. This is illustrated on the MDN. 
https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toDataURL
